[2025-02-24T05:05:18.744+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T05:05:18.769+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_1.first_task scheduled__2025-02-09T00:00:00+00:00 [queued]>
[2025-02-24T05:05:18.781+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_1.first_task scheduled__2025-02-09T00:00:00+00:00 [queued]>
[2025-02-24T05:05:18.782+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T05:05:18.895+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-02-09 00:00:00+00:00
[2025-02-24T05:05:18.903+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1880) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T05:05:18.905+0000] {standard_task_runner.py:72} INFO - Started process 1895 to run task
[2025-02-24T05:05:18.908+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_1', 'first_task', 'scheduled__2025-02-09T00:00:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpkrxjww3s']
[2025-02-24T05:05:18.910+0000] {standard_task_runner.py:105} INFO - Job 135: Subtask first_task
[2025-02-24T05:05:18.981+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_1.first_task scheduled__2025-02-09T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T05:05:19.074+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag_1' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-09T00:00:00+00:00'
[2025-02-24T05:05:19.077+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T05:05:19.091+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T05:05:19.092+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world! the first task is defined!!!']
[2025-02-24T05:05:19.103+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T05:05:19.104+0000] {subprocess.py:106} INFO - hello world! the first task is defined!!!
[2025-02-24T05:05:19.104+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T05:05:19.136+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T05:05:19.137+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag_1, task_id=first_task, run_id=scheduled__2025-02-09T00:00:00+00:00, execution_date=20250209T000000, start_date=20250224T050518, end_date=20250224T050519
[2025-02-24T05:05:19.204+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T05:05:19.219+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
